sdk/highlight-remix/package.json

```diff
@@ -0,0 +1,50 @@
+{
+	"name": "@highlight-run/remix",
+	"version": "0.0.0",
```

Comment:
not sure what the convention is here, but i've only seen `0.0.0` for unpublishable packages. should this be `0.1.0`?

Reply:
Fixing.